safetypy/safetypy.py

```diff
@@ -365,6 +366,36 @@ def get_web_report(self, audit_id):
         else:
             return None
 
+    def get_audit_actions(self, date_modified, offset=0):
+        """
+        Get all actions created after a specified date. If the number of action is more than 100, this function will
+        page until it has collected all actions up to 5000 actions.
+        :param date_modified:    ISO formatted date/time string. Only actions created after this date are are returned.
```

Comment:
Is this local or utc time?

Reply:
UTC

Reply:
but from the perspective of this function, it doesn't care if the time is local or UTC -- it just accepts an ISO formatted string to use in the search.

Reply:
It would be helpful for the caller of the function to know what iAuditor expects there from a functional perspective.